lib/azure_jwt_auth/provider.rb

```diff
@@ -1,4 +1,4 @@
-require 'net/http'
+require 'net/https'
```

Comment:
This is not needed. From [Https section of documentation](https://ruby-doc.org/stdlib-2.6.5/libdoc/net/http/rdoc/Net/HTTP.html):

```
In previous versions of Ruby you would need to require 'net/https' to use HTTPS. This is no longer true.
```